packages/web-components/src/components/quote/__stories__/quote.stories.ts

```diff
@@ -0,0 +1,81 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { boolean, select } from '@storybook/addon-knobs';
+import { html } from 'lit-element';
+import ArrowRight20 from 'carbon-web-components/es/icons/arrow--right/20.js';
+import { QUOTE_TYPES } from '../quote';
+import '../../link-with-icon/link-with-icon';
+import textNullable from '../../../../.storybook/knob-text-nullable';
+
+import readme from './README.stories.mdx';
+
+export const Default = ({ parameters }) => {
+  const { copy, quoteMark, sourceHeading, sourceCopy, sourceBottomCopy, inverse } = parameters?.props?.Quote ?? {};
+  return html`
+    <dds-quote .inverse=${inverse} mark-type="${quoteMark}">
```

Comment:
I think we should demonstrate in the story that `inverse` attribute is a boolean attribute:

```suggestion
    <dds-quote ?inverse="${inverse}" mark-type="${quoteMark}">
```